app/uk/gov/hmrc/agentepayeregistration/repository/AgentEpayeRegistrationRepository.scala

```diff
@@ -19,24 +19,24 @@ package uk.gov.hmrc.agentepayeregistration.repository
 import javax.inject.{Inject, Singleton}
 import org.joda.time.{DateTime, DateTimeZone}
 import play.api.Logger
-import play.api.libs.json._
```

Comment:
And the findStaleReferenceFields function + removeRedundantFields. Not necessarily part of this story but since you partially deleted the other files. May aswell finish it off.